src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory_Methods.cs

```diff
@@ -345,7 +345,7 @@ private static ArgumentKind GetArgumentKind(BoundExpression argument, ref BitVec
             }
             else if (argument.IsParamsCollection)
             {
-                argumentKind = ArgumentKind.ParamArray;
+                argumentKind = argument.Type?.IsSZArray() == true ? ArgumentKind.ParamArray : ArgumentKind.ParamCollection;
```

Comment:
I thought that `IsParamsCollection` would only be true if it was actually a params collection, and `IsParamsArray` would be true in the other case, with `IsParams` being the catch-all. Is that just not implemented yet?

Reply:
> I thought that `IsParamsCollection` would only be true if it was actually a params collection, and `IsParamsArray` would be true in the other case, with `IsParams` being the catch-all. Is that just not implemented yet?

This is not an IOperation API. Bound nodes do not have `IsParamsArray` or `IsParams` properties. The internal `IsParamsCollection` property covers both cases.

Reply:
I see. It may be worth a refactoring to standardize them at some point then.

Reply:
> It may be worth a refactoring to standardize them at some point then.

I do not think it is needed.